assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -127,16 +112,18 @@ function Toggle({
       variant={BUTTON_VARIANTS.RECTANGLE}
       size={BUTTON_SIZES.MEDIUM}
     >
-      <HelpIcon />
-      <Label>{__('Help', 'web-stories')}</Label>
-      {hasNotifications && (
-        <NotificationWrapper>
-          <NotificationBubble notificationCount={notificationCount} />
-        </NotificationWrapper>
-      )}
-      <Icon hasNotifications={hasNotifications} isOpen={isOpen}>
-        <ChevronIcon />
-      </Icon>
+      <Label>{__('Help Center', 'web-stories')}</Label>
+      <IconWrapper>
+        {hasNotifications ? (
+          <NotificationWrapper>
+            <NotificationBubble notificationCount={notificationCount} />
+          </NotificationWrapper>
+        ) : (
+          <Icon isOpen={isOpen}>
+            <ChevronIcon />
+          </Icon>
```

Comment:
So now the chevron only shows after the notifications are gone. Is that intended?
![image](https://user-images.githubusercontent.com/22185279/107675525-66f0a900-6c55-11eb-8110-2f52b150e7ce.png)

Reply:
![image](https://user-images.githubusercontent.com/22185279/107675678-8c7db280-6c55-11eb-928a-5aaa9a7ad94e.png)

Reply:
it is, designs were updated very recently! https://xwp.slack.com/archives/C01ESBB8ZN1/p1612994877142300

Reply:
Yes, the designs have been updated to [this](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=5663%3A3546). Isn't it lovely with a moving target? 😁 

But I know see, that it has actually updated again since this implementation for the narrow viewport. Here "Help Center" is replaced by just the question mark icon followed by either the unread count or the chevron.

Reply:
oh heck, i thought it was that on narrow viewport it's just 'help' + the chevron or badge depending on if there's unread tips? Either way, this update's not going in until after the release is cut so if it's off a little bit when this is merged it's ok because we can just have a folllow up ticket/pr to actually address that. I feel like adjusting this FTUE here is just scope creep.

Reply:
This will actually go in, as this mainly updates icons in the design system, and only very few icons in the actual editor - and none of those are changed visually, they're merely updated in sizing and bounding box.

Reply:
Implemented the new designs in 629038057. Feel free to take them for a spin - I think it looks pretty good. Also, see follow-up discussion [here](https://xwp.slack.com/archives/C01ESBB8ZN1/p1613098933195200).